### DISCUSSION TITLE
Dark overlay / fade out screen when showing Ulauncher

### DISCUSSION THREAD

**Discussion by shinyvision** [Ulauncher/Ulauncher#894] [Ideas]
I think it would be helpful if the Ulauncher could darken the screen behind the launcher to make it more clear the launcher is opened. Several times I've pressed my super key multiple times thinking the launcher wasn't there when in fact I wasn't looking closely enough.

I don't think this feature exists in the program yet, and I think it would be useful if users could check a checkbox to indicate they would like to darken the screen when the launcher is opened.

I have no idea how easy something like this would be to implement (I don't normally work on desktop apps), but if this is something more people want to see added, I could take a shot at contributing to implement this.

**Comment by friday** [comment]
Thank you for the offer to help. I personally wouldn't want that behavior, nor to maintain that solution so it's working or at least not breaking other behavior on all desktops, including tiling desktops and desktops without a compositor. I'm not aware of any good ways to achieve this, if it's even possible. It's a very popular thing on the web, but I don't think this translates to desktop apps.

* What if you have two screens? Should it blur/overlay both or just the one?
* What if you want to use the Ulauncher calculator and read from another window so you can use that as data in your calculation (I do this sometimes)?

I will leave the issue open to see if others want this or have other related input though.
